app/routes/git_tokens.py

```
@@ -39,6 +98,11 @@ def format_token_response(token_data: Dict[str, Any]) -> Dict[str, Any]:
     """
 
     if token_data and token_data.get("token_value"):
+        try:
+            decrypt_data = EncryptionHelper.decrypt(token_data.get("token_value", ""))
```

Ruff (0.11.9)

Local variable `decrypt_data` is assigned to but never used

Remove assignment to unused variable `decrypt_data`

(F841)